BirthdayBot/Cogs/BirthdayChecker.py

```diff
@@ -305,6 +305,56 @@ async def monthceleb(self, ctx):
         await ctx.send(embed=embed)
         CommandCounter.incrementCommand("monthceleb")
 
+    @commands.hybrid_command(
+        name="me",
+        description="Displays user infromation along with a random celebrity that shares the same brithday with the user.",
+    )
+    async def me(self, ctx):
+        if DiscordUser.does_user_exist(discord_id=ctx.author.id):
+            user = DiscordUser.get(discord_id=ctx.author.id)
+            userBday = user.birthday
+            with session_scope() as session:
+                celebBday = (
```

Comment:
Can't we use getAllBirthdays(celeb=True, date=user.birthday) here? 

That would make it all line and utilizes that function we developed earlier.

Reply:
`        monthBdays = BirthdayChecker.getAllBirthdays(celeb=True, checks_only_month=True)`

From above ^

Reply:
fixed this